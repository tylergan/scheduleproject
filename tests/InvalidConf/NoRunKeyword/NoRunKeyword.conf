on Tuesday at 1100 /bin/date