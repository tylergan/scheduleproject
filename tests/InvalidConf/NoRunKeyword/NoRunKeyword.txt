Testing that runner.py outputs the correct output to STDERR when there is:
    1) No run keyword located in the command line of the config file
