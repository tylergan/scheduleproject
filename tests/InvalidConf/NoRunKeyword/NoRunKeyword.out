error in configuration: on Tuesday at 1100 /bin/date
